src/components/home/about_section.js

```diff
@@ -43,8 +43,8 @@ const AboutSection = ({ planetMorph }) => (
           </div>
           <p className={styles.text}>
             <Text color="white">
-              We'll challenge you to think further and help you do the heavy
-              lifting of shaping your product's development.
+              We&apos;ll challenge you to think further and help you do the
```

Comment:
I question this linter setting. What's the advantage of these replacements when using UTF-8?

Reply:
Basically, this, to avoid some weird bugs where people leave some random Javascript characters layin around: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md

Reply:
Dumb rule but ok...